styles.js

```diff
@@ -12,6 +15,7 @@ export default createGlobalStyle`
     background: rgb(44, 150, 164);
     background-attachment: fixed;
     background-color: #584849;
+    font-family: ${grandstander.style.fontFamily};
   }
 
   h1, h2, h3, h4, h5, h6, p, ul, ol {
```

Comment:
if you want the following rules to apply to all elements, you could pass them to the * above